skyrl-train/skyrl_train/workers/worker.py

```diff
@@ -660,15 +692,15 @@ def ppo_train(self, train_data: TrainingInputBatch) -> TrainingOutputBatch:
         # not needed beyond status logging
         all_metrics.pop("response_length", None)
 
-        status_mean = reduce_metrics(all_metrics)
-        status_mean["policy_update_steps"] = policy_update_steps / accumulation_steps
+        status_mean = reduce_metrics(all_metrics, dataloader.total_batch_size * self.cfg.trainer.update_epochs_per_batch * self._world_size)
```

Comment:
Pretty sure this is right but worth taking a double click at this to make sure this weighted average works in all cases.